.github/pull_request_assignment.yml

```diff
@@ -105,32 +116,24 @@
    
       
 - rule:
-    paths: "specification/@(providerhub|sql|appplatform|deviceupdate|servicelinker|imagebuilder|throttling|secretmanagementtestsvc|containerservice|azure-kusto)/**"
-    reviewers:
-      - raych1
-      
-- rule:
-    paths: "specification/voiceservices/**"
```

Comment:
Did you intentionally remove voiceservices?

Reply:
It would be assigned randomly.